ddf-test/src/main/scala/io/ddf/test/it/etl/TransformationHandlerBaseSuite.scala

```diff
@@ -55,22 +55,251 @@ trait TransformationHandlerBaseSuite extends BaseSuite with Matchers {
+    scaledDDF.getNumColumns should be(11)
+
+    val scaledDDFSummary: Array[Summary] = scaledDDF.getSummary
+    scaledDDFSummary(0).min should be(0)
+    scaledDDFSummary(0).max should be(1)
+    scaledDDFSummary(1).min should be(0)
+    scaledDDFSummary(1).max should be(1)
+    scaledDDFSummary(4).min should be(0)
+    scaledDDFSummary(4).max should be(1)
+    // The following column has min == max, hence should not be scaled
+    scaledDDFSummary(2).min should be(3)
+    scaledDDFSummary(2).max should be(3)
+    scaledDDFSummary(3).min should be(4)
+    scaledDDFSummary(3).max should be(4)
+    // Not numeric columns
+    scaledDDFSummary(8) should be(null)
+    scaledDDFSummary(10) should be(null)
+
+    val data = scaledDDF.VIEWS.head(scaledDDF.getNumRows.toInt)
+    val arr = data.map{str => str.split("\\t")}
+
+    arr.get(0)(0).toDouble should be(0)
+    arr.get(1)(0).toDouble should be(0.5)
+    arr.get(2)(0).toDouble should be(1)
+
+    arr.get(0)(2).toDouble should be(3)
+    arr.get(1)(2).toDouble should be(3)
+    arr.get(2)(2).toDouble should be(3)
+
+    arr.get(0)(4).toDouble should be((2003 - originalDDFSummary(4).min) / (originalDDFSummary(4).max-originalDDFSummary(4).min))
+    arr.get(1)(4).toDouble should be((754 - originalDDFSummary(4).min) / (originalDDFSummary(4).max-originalDDFSummary(4).min))
+    arr.get(2)(4).toDouble should be((628 - originalDDFSummary(4).min) / (originalDDFSummary(4).max-originalDDFSummary(4).min))
+
+    arr.get(0)(8) should be("WN")
+    arr.get(1)(8) should be("WN")
+    arr.get(2)(8) should be("WN")
   }
 
-  ignore("transform scale standard") {
-    val relevantData: DDF = loadAirlineDDF().VIEWS.project("Year", "Month", "DayofMonth", "DayofWeek", "DepTime", "CRSDepTime", "ArrTime", "CRSArrTime")
-    val newDDF: DDF = relevantData.Transform.transformScaleStandard()
-    newDDF.getNumRows should be(31)
-    newDDF.getSummary.length should be(8)
+  test("transform scale min max subset of columns") {
+    val originalDDF: DDF = loadAirlineDDF().VIEWS.project("Year", "Month", "DayofMonth", "DayofWeek", "DepTime",
+      "CRSDepTime", "ArrTime", "CRSArrTime", "UniqueCarrier", "FlightNum", "TailNum")
+    val originalDDFSummary: Array[Summary] = originalDDF.getSummary
+
+    val columnsToScale = List("Year", "DayofMonth", "TailNum", "UnknownColumn", null, "DayofMonth")
+    val scaledDDF: DDF = originalDDF.Transform.transformScaleMinMax(columnsToScale, false)
+    scaledDDF should not be null
+    scaledDDF.getNumRows should be(31)
+    scaledDDF.getNumColumns should be(11)
+
+    val scaledDDFSummary: Array[Summary] = scaledDDF.getSummary
+    scaledDDFSummary(0).min should be(0)
+    scaledDDFSummary(0).max should be(1)
+    scaledDDFSummary(1).min should be(1)
+    scaledDDFSummary(1).max should be(11)
+    scaledDDFSummary(4).min should be(617)
+    scaledDDFSummary(4).max should be(2107)
+    // The following column has min == max, hence should not be scaled
```

Comment:
This is new logic added to ignore scaling if min==max as this would cause a divide-by-zero exception. At the client side, the documentation also mentions this but we don't have an explicit check in PE. (Same idea for scale standard if stdev == 0)